python/py-testtools/Portfile

```diff
@@ -24,11 +24,11 @@ homepage            https://github.com/testing-cabal/testtools
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
 
-checksums           rmd160  bd9c1f31612d7dc7c9fb1bc657167ef1109cec02 \
-                    sha256  5827ec6cf8233e0f29f51025addd713ca010061204fdea77484a2934690a0559 \
-                    size    231559
+checksums           rmd160  203aa69815a8b75b699f6f588431330aa2d3f072 \
+                    sha256  57c13433d94f9ffde3be6534177d10fb0c1507cc499319128958ca91a65cb23f \
+                    size    235399
 
-python.versions     27 37
+python.versions     27 37 38 39
```

Comment:
this version doesn't support PY27 anymore so you need to pin it to the latest supported version which according to upstream is 2.4.0. Please check whether the package got built already for PY27 (but likely doesn't work), if that's the case you'll need to increase the `epoch`. Please make sure to check these thing when updating the port / before merging.

Reply:
Thanks for the heads-up Renee, I'll fix this for Steve.

Reply:
Fixed in commit: https://github.com/macports/macports-ports/commit/eb5b4968f3faedc9e3ffb7efb087e8848a5528b8